src/System.Runtime/tests/System/Runtime/Tests1.7.cs

```diff
@@ -0,0 +1,33 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
In fact, looks like you are already adding that file in the CompilerServices folder, should we just move these tests over there?

---
In reply to: [81802069](https://github.com/dotnet/corefx/pull/12337#discussion_r81802069) [](ancestors = 81802069)